reset.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# The purpose of this script is to make it easy to reset a local onpremise
+# install to a clean state, optionally installing a particular version/ref.
+
+set -euo pipefail
+
+if [ -n "${DEBUG:-}" ]; then
+  set -x
+fi
+
+cd "$(dirname $0)"
+
+
+function confirm () {
+  read -p "$1 [y/n] " confirmation
+  if [ "$confirmation" != "y" ]; then
+    echo "Canceled. 😅"
+    exit
+  fi
+}
+
+# Make sure they mean it.
+confirm "☠️  Warning! 😳 This is highly destructive! 😱 Are you sure you wish to proceed?"
+echo "Okay ... good luck! 😰"
+
+# Hit the reset button.
+docker compose down --volumes
```

Comment:
Let's make it even more destructive:
```suggestion
docker compose down -v --remove-orphans --rmi local
```

Reply:
> `--remove-orphans`

[Docs](https://docs.docker.com/engine/reference/commandline/compose_down/):

> Remove containers for services not defined in the Compose file.

If I have other non-Sentry containers on my laptop I don't want those clobbered here, do I?

Reply:
That's a bit of a misrepresentation. They are trying to say containers under the same "project" umbrella. It wouldn't clean up all the containers on your laptop. `docker container prune` exists for that :)

Reply:
`--rmi local` in 387a72953a631ce0fd5540e0deb3cc299c48f8de.